### PR TITLE
Nav buttons

### PR DESCRIPTION
**Pull request by jamesooden**
This will help manage and contain information for the wizards cancel, back, and next buttons and the additional buttons that come when you select cancel.   I have tested this, but I haven't hooked it in yet.   The next step will be to make it be contained by the Base class.   BTW, that is why this does not inherit from the Base class, because it must be contained by it, and I didn't want to have the circular dependency.

### CODE REVIEW

File: pages/wizard/regions/navigation_buttons.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        '//button[contains(.,"Exit and Save")]'
+    )
+    _continue_working_loc = (
+        By.XPATH,
+        '//button[contains(.,"Continue Working")]'
+    )
+
+    # properties
+    @property
+    def cancelBtn(self):
```
Comment (blrm) [inline pages/wizard/regions/navigation_buttons.py]: I think cancelBtn, backBtn, and nextBtn methods should be in snake_case to be consistent with the rest of the methods in the file.

Comment (jamesooden) [inline pages/wizard/regions/navigation_buttons.py]: Right...this is the sucky part of this...I moved them to a different file so the previous fix just kind of goes way.  On it.

Comment (jamesooden) [inline pages/wizard/regions/navigation_buttons.py]: :heavy_check_mark:

**Comment by blrm** [comment]
:+1: Looks great!

**Pull request merged**